packaging/cockpit-starter-kit.spec.in

```diff
@@ -7,7 +7,7 @@ License: LGPLv2+
 Source0: https://github.com/cockpit-project/starter-kit/releases/download/%{version}/%{name}-%{version}.tar.xz
 Source1: https://github.com/cockpit-project/starter-kit/releases/download/%{version}/%{name}-node-%{version}.tar.xz
 BuildArch: noarch
-BuildRequires:  nodejs
+BuildRequires:  nodejs-devel
```

Comment:
While we are at it, let's fix this double space and thus the alignment. :grin:

Reply:
Thanks, fixed locally -- will push when the bots PR is merged, refreshes included.

Reply:
see also the [c9s build failure](https://copr.fedorainfracloud.org/coprs/packit/cockpit-project-starter-kit-573/build/4383847/) -- `nodejs-devel` isn't a thing there. There is no technical reason to actually b-r -devel, nodejs is really enough. It's just about the Fedora packaging policy. So let's conditionalize this on RHEL?

Reply:
Indeed, it does not even exist in CRB :cry:

OK, added an additional commit to limit it on Fedora.